### PR TITLE
Always run standalone tests

### PR DESCRIPTION
**Pull request by carmocca**
## What does this PR do?

Addresses https://github.com/Lightning-AI/lightning/pull/16697#discussion_r1101699402

cc @carmocca @akihironitta @borda

### CODE REVIEW

**Review by akihironitta — APPROVED** [review]

File: .azure/gpu-tests-fabric.yml [hunk 1]
```diff
@@ -126,7 +126,6 @@ jobs:
       env:
         PL_RUN_CUDA_TESTS: "1"
         PL_STANDALONE_TESTS_SOURCE: "lightning_fabric"
```
Comment (Borda) [inline .azure/gpu-tests-fabric.yml:128]:
```suggestion
        PL_STANDALONE_TESTS_SOURCE: $(COVERAGE_SCOPE)
```

File: .azure/gpu-tests-pytorch.yml [hunk 2]
```diff
@@ -163,7 +163,6 @@ jobs:
         PL_RUN_CUDA_TESTS: "1"
         PL_STANDALONE_TESTS_SOURCE: "pytorch_lightning"
```
Comment (Borda) [inline .azure/gpu-tests-pytorch.yml:164]:
```suggestion
        PL_STANDALONE_TESTS_SOURCE: $(COVERAGE_SCOPE)
```

**Review by Borda — APPROVED** [review]

File: .azure/gpu-tests-pytorch.yml [hunk 3]
```diff
@@ -161,9 +161,8 @@ jobs:
       env:
         PL_USE_MOCKED_MNIST: "1"
         PL_RUN_CUDA_TESTS: "1"
-        PL_STANDALONE_TESTS_SOURCE: "pytorch_lightning"
+        PL_STANDALONE_TESTS_SOURCE: $(COVERAGE_SCOPE)
```
Comment (carmocca) [inline .azure/gpu-tests-pytorch.yml:164]: @Borda This doesn't work: it's passing:

```$(' python -c ''\''n' = '"lightning"' ';' 'print(dict(fabric="lightning_fabric").get(n,' 'n))'\''' ')```

Comment (Borda) [reply .azure/gpu-tests-pytorch.yml]: shall be resolved in [0009239](https://github.com/Lightning-AI/lightning/pull/16705/commits/00092390cfe4156ced7277b040f799f6a859983b)

**Review by justusschock — APPROVED** [review]

**Comment by Borda** [comment]
The failing error seems to be similar to #16706

**Pull request merged**